### PR TITLE
feat/optimism [TKR-280]

### PR DESCRIPTION
**Pull request by shawnxin89**
## Description

[TKR-280]

Optimism support:

Add uniswap V3 

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/contract-addresses/addresses.json [hunk 1]
```diff
@@ -460,5 +460,89 @@
...
+        "erc20BridgeSampler": "0x0000000000000000000000000000000000000000",
+        "chaiBridge": "0x0000000000000000000000000000000000000000",
+        "dydxBridge": "0x0000000000000000000000000000000000000000",
+        "godsUnchainedValidator": "0x0000000000000000000000000000000000000000",
+        "broker": "0x0000000000000000000000000000000000000000",
+        "chainlinkStopLimit": "0x0000000000000000000000000000000000000000",
+        "maximumGasPrice": "0x0000000000000000000000000000000000000000",
+        "dexForwarderBridge": "0x0000000000000000000000000000000000000000",
+        "exchangeProxyGovernor": "0x6d506b2847df0c6f04d2628da1adaf4d8fb2e81b",
+        "exchangeProxy": "0xdef1abe32c034e558cdd535791643c58a13acc10",
```
Comment (kimpers) [inline packages/contract-addresses/addresses.json:493]: EP verification on Optimistic Etherscan looks a bit funky https://optimistic.etherscan.io/address/0xdef1abe32c034e558cdd535791643c58a13acc10#code. Can we still verify it properly?

File: packages/contract-addresses/addresses.json [hunk 2]
```diff
@@ -460,5 +460,89 @@
...
+        "exchangeProxyFlashWallet": "0x0000000000000000000000000000000000000000",
+        "exchangeProxyLiquidityProviderSandbox": "0x0000000000000000000000000000000000000000",
+        "zrxTreasury": "0x0000000000000000000000000000000000000000",
+        "transformers": {
+            "wethTransformer": "0xCF920E54d6342830E81b43cdAbEc0712DC19950C",
+            "payTakerTransformer": "0x9b136b3B41cdbFeF2d5Dc9B2eDF4D62eC7423e73",
+            "affiliateFeeTransformer": "0x3028380B92a6615Eec58e6aA784DC5B7E6C89A1F",
+            "fillQuoteTransformer": "0x4b545D3fC14a28f1ecbe57Ebb43705799F176579",
+            "positiveSlippageFeeTransformer": "0x475064343BA0083A1610BC86398cb6b27ac2758c"
+        }
```
Comment (kimpers) [inline packages/contract-addresses/addresses.json:546]: I could be wrong but I don't think we use checksummed addresses in this file

**Review by kimpers — CHANGES_REQUESTED** [review]

File: packages/contract-addresses/addresses.json [hunk 3]
```diff
@@ -460,5 +460,89 @@
...
+        "zrxTreasury": "0x0000000000000000000000000000000000000000",
+        "transformers": {
+            "wethTransformer": "0x02ce7af6520e2862f961f5d7eda746642865179c",
+            "payTakerTransformer": "0x085d10a34f14f6a631ea8ff7d016782ee3ffaa11",
+            "affiliateFeeTransformer": "0x55cf1d7535250db75bf0190493f55781ee583553",
+            "fillQuoteTransformer": "0x3543ef833d28b7e983c293856561f21a7f089f1d",
+            "positiveSlippageFeeTransformer": "0xb11e14565dfbeb702dea9bc0cb47f1a8b32f4783"
+        }
+    },
+    "69": {
```
Comment (dorothy-zbornak) [inline packages/contract-addresses/addresses.json:506]: Do we really need the testnet as an entry? Can we remove this?

**Review by dorothy-zbornak — APPROVED** [review]
:+1: :eye: :lips: :eye: :+1:

**Review by kimpers — APPROVED** [review]
Awesome work! 🙏

**Pull request merged**